packages/story-editor/src/components/canvas/singleSelectionMoveable/index.js

```diff
@@ -53,30 +53,20 @@ function SingleSelectionMoveable({
   const moveable = useRef(null);
   const [isDragging, setIsDragging] = useState(false);
 
-  const { nodesById } = useCanvas(({ state: { nodesById } }) => ({
-    nodesById,
-  }));
-  const { getBox } = useUnits(({ actions: { getBox } }) => ({
-    getBox,
-  }));
-  const {
-    actions: { pushTransform },
-  } = useTransform();
-  const { zoomSetting, scrollLeft, scrollTop } = useLayout(
-    ({ state: { zoomSetting, scrollLeft, scrollTop } }) => ({
-      zoomSetting,
-      scrollLeft,
-      scrollTop,
-    })
-  );
+  const nodesById = useCanvas((v) => v.state.nodesById);
+  const getBox = useUnits((v) => v.actions.getBox);
+  const pushTransform = useTransform((v) => v.actions.pushTransform);
+  const zoomSetting = useLayout((v) => v.state.zoomSetting);
+  const scrollLeft = useLayout((v) => v.state.scrollLeft);
+  const scrollTop = useLayout((v) => v.state.scrollTop);
```

Comment:
Maybe leave a comment to explain why it's better that `zoomSetting`, `scrollLeft`, `scrollTop` are all using `useLayout` separately instead of just using `useLayout(({ state })` for all.

Reply:
Yup. put an answer there too. Will post something in the web-stories slack channel about the update too cus I've been getting a lot of questions about it:
https://github.com/google/web-stories-wp/pull/10051/files#r778230330